tools/repl/repl.cpp

```diff
@@ -42,6 +45,9 @@
 
 #define C_STRING(x) #x
 
+#define HISTORY_FILE ".hermes_history"
+#define HISTORY_MAX_ENTRIES 500
```

Comment:
Can we use `static const` variables here instead of preprocessor macros? They have the advantage of having proper typing.

Reply:
Done